track_analyzing/track_analyzer/track_analyzer.cpp

```diff
@@ -30,6 +30,7 @@ DEFINE_string_ext(cmd, "",
                   "match - based on raw logs gathers points to tracks and matches them to "
                   "features. To use the tool raw logs should be taken from \"trafin\" production "
                   "project in gz files and extracted.\n"
+                  "match_dir - the same as match but applies to the directory with raw logs in gz format.\n"
```

Comment:
Probably it's worth saying that it's more efficient option in case of multithreading CPU.